Update interfaces/layout-instability.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/633a365/ed/idl/layout-instability.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/669029377

stephenmcgruer commented:
This will need the idlharness file modified for the new interface, plus it also contains a spec error. Opened https://github.com/WICG/layout-instability/issues/40

autofoolip committed:
Update interfaces/layout-instability.idl

Source: https://github.com/tidoust/reffy-reports/blob/633a365/ed/idl/layout-instability.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/669029377

foolip commented:
The `FrozenArray` fix from https://github.com/WICG/layout-instability/pull/41 has reached this PR now.
